great_expectations/data_context/data_context.py

```diff
@@ -3754,16 +3905,119 @@ def scaffold_notebooks(cls, base_dir):
                 destination_path = os.path.join(subdir_path, notebook_name)
                 shutil.copyfile(notebook, destination_path)
 
-    def __init__(self, context_root_dir=None, runtime_environment=None):
+    @classmethod
+    def _get_ge_cloud_config_dict(
+        cls,
+        ge_cloud_base_url: Optional[str] = None,
+        ge_cloud_account_id: Optional[str] = None,
+        ge_cloud_access_token: Optional[str] = None,
+    ):
+        ge_cloud_base_url = (
+            ge_cloud_base_url
+            or super()._get_global_config_value(
+                environment_variable="GE_CLOUD_BASE_URL",
+                conf_file_section="ge_cloud_config",
+                conf_file_option="base_url",
+            )
+            or "https://app.greatexpectations.io/"
```

Comment:
Just curious : How is `GE_CLOUD_BASE_URL` going to be configured for the Anthony and Sarah milestones? is `https://app.greatexpectations.io` part of either milestone?

Reply:
For the Anthony milestone, we'll just provide him with an appropriate `great_expectations.conf`, which would include the base url (in his case, https://app.dev.greatexpectations.io).